apps/examples/testcase/ta_tc/media/itc/itc_media_mediaplayer.cpp

```diff
@@ -31,16 +31,21 @@
  ****************************************************************************/
 static const char *file_path = "/mnt/fileinputdatasource.raw";
 static const char *test_data = "dummydata";
-static int g_flag=0;
+static int g_flag = 0;
 
 static void SetUp(void)
 {
-	FILE *fp;
-	fp = fopen(file_path, "w");
-	TC_ASSERT_NEQ("fopen", fp, NULL);
-	TC_ASSERT_NEQ_CLEANUP("fputs", fputs(test_data, fp), EOF, fclose(fp));
-	TC_ASSERT_EQ("fclose", fclose(fp), OK);
-	g_flag = 1;
+	FILE *fp = fopen(file_path, "w");
+	if (fp != NULL) {
+		int ret = fputs("dummydata", fp);
```

Comment:
how about 
`
int ret = fputs(test_data, fp);
if (ret != (int)strlen(test_data)) {
	printf("fail to fputs\n");
}
`